source/localizable/smart_cart/webhooks.html.md.erb

```diff
@@ -0,0 +1,160 @@
+---
+parent: smart_cart
+title: webhooks
+---
+
+<%= partial 'partials/page_locales' %>
+
+# Webhooks <%= edit_link %>
+
+The Smart Cart webhooks allow Skroutz merchants to automatically integrate their
+Smart Cart orders information with their internal tools.
+
+<%= partial 'partials/toc' %>
+
+## Setup
+
+In order to be able to use the webhook service, a webhook url should be registered
+by the merchant from within
+<%= link_to 'Smart Cart settings',
+            "https://#{settings.merchants_domain}/merchants/account/shop/ecommerce_details",
+            target: '_blank' %>. You may find a high level overview of the service and
+descriptive guides for its enabling in the
+<%= link_to 'merchants support guidelines',
+            "https://#{settings.merchants_domain}/merchants/support/guidelines/smart_cart_webhooks",
+            target: '_blank' %> page.
+The webhook url should match the shop's domain and it should follow secure <strong>HTTP (HTTPS)</strong>,
+as we do not support requests over an insecure protocol. We provide a valid webhook
+url for a given shop link in the following table:
+
+##### Valid webhook url
+
+Shop URL | Webhook URL
+-------- | ----------
+`https://shop.gr` | `https://shop.gr/webhook_url`
+
+## Webhook events
+
+A webhook request is sent to the predefined webhook url once an event has been triggered.
+As of right now, the only supported webhook event is the placement of a
+<strong>new Smart Cart order</strong>. We provide an overview of the request
+headers and payload to be sent by our side to the webhook url.
+
+### New order
+
+A <strong>new order</strong> webhook request is triggered upon the placement of a new order.
+Any failed request should be retried in a later time. It should be expected that a request
+concerning the same order would be performed up to 3 times in total, every 15 minutes.
+We also provide the possibility of manually sending / retrying a new order webhook request
+from within the order page, however the expected request would be the same in all the above
+scenarios.
+
+##### Request
+<pre class="terminal">
+POST {webhook_url}
+</pre>
+
+##### Expected request response
+
+<pre class="terminal">
+200
+</pre>
+
+##### Request headers
+
+Header         | Value
+-------------- | -----
+`Content-Type` | `application/json: charset=utf-8`
+`User-Agent`   | `Skroutz OrderNotifier v1`
+
+##### Request payload
+
+Name         | Type   | Description
```

Comment:
We could add a "Value" column to list the possible values of `event_type` (only one for now), like this: http://developer.skroutz.gr/api/v3/category/#ordering